featuretools/entityset/entityset.py

```diff
@@ -801,6 +802,8 @@ def normalize_entity(self, base_entity_id, new_entity_id, index,
         self.delete_entity_variables(base_entity_id, additional_variables)
 
         new_entity = self.entity_stores[new_entity_id]
+        if make_secondary_time_index:
+            new_entity.secondary_time_index = {secondary_time_index: [secondary_time_index]}
```

Comment:
This should follow the logic in `BaseEntity.__init__()`. You'll want to append all the variables included in the values of the make_secondary_time_index dict here, except for the time index itself whose name changed and is already in the list. To be complete, we should probably allow for multiple `secondary_time_index`'s too, right? The dictionary format allows an arbitrary number of them. In this case the `new_entity_secondary_time_index` parameter to `normalize_entity()` would need to be a dictionary mapping the column names in the base entity to the ones you want in the new entity.

Reply:
So, if I'm understanding correctly, these two lines should be changed to
```
new_entity.secondary_time_index = secondary_time_index or {}
for ti, cols in new_entity.secondary_time_index.items():
    if ti not in cols:
        cols.append(ti)
```
It seems like multiple secondary time index is also asserted against with 
```
            assert len(make_secondary_time_index) == 1, "Can only provide 1 secondary time index"
``` 
in `entityset.py`.